Source/ACE.Database/Database.cs

```diff
@@ -78,8 +84,8 @@ public void AddPreparedStatement<T>(T id, params object[] parameters)
                 }
                 catch (MySqlException transactionException)
                 {
-                    Console.WriteLine($"An exception occured while commiting a transaction of {queries.Count} queries, a rollback will be performed!");
-                    Console.WriteLine($"Exception: {transactionException.Message}");
+                    log.Error($"An exception occured while commiting a transaction of {queries.Count} queries, a rollback will be performed!");
```

Comment:
Should this be set to Error channel? I believe it's more relevant to capture these exceptions as errors, if different logging levels are being utilized.